battleship.py

```diff
@@ -62,7 +62,13 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    list=[]

```

Comment:
Rename list variable as board or grid, so that it is more suitable according to the function.

Reply:
@meghamittall make the requested changes, then only I can review it again.

Reply:
@Pragya28 I made changes as you asked me to do. You can please review it now.